Well this one failed with out of memory errors when we tried to open a subprocess

```
=================================== FAILURES ===================================
_____________________________ test_initial_warning _____________________________
[gw0] linux -- Python 3.6.8 /home/travis/miniconda3/envs/pandas-dev/bin/python
    def test_initial_warning():
        code = (
            "import pandas as pd; import matplotlib.pyplot as plt; "
            "s = pd.Series(1, pd.date_range('2000', periods=12)); "
            "fig, ax = plt.subplots(); "
            "ax.plot(s.index, s.values)"
        )
        call = [sys.executable, "-c", code]
>       out = subprocess.check_output(call, stderr=subprocess.STDOUT).decode()

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,3 @@
-sudo: false
 language: python
 python: 3.5
 

--- a/ci/deps/azure-36-locale.yaml
+++ b/ci/deps/azure-36-locale.yaml
@@ -7,7 +7,6 @@ dependencies:
   - bottleneck=1.2.*
   - cython=0.28.2
   - lxml
-  - matplotlib=2.2.2
   - numpy=1.14.*
   - openpyxl=2.4.8
   - python-dateutil

--- a/ci/deps/travis-36-cov.yaml
+++ b/ci/deps/travis-36-cov.yaml
@@ -11,7 +11,6 @@ dependencies:
   - gcsfs
   - geopandas
   - html5lib
-  - matplotlib
   - moto
   - nomkl
   - numexpr

